File: banda/IvanRC-banda-Java/src/main/java/modelsBandaMusicos/Cantante.java
package modelsBandaMusicos;

public class Cantante extends Musico implements ICantante {

    String nombre;
    Double salario = 1500.00;
    int añosExperiencia;
    String tono;

    public Cantante(String nombre, int añosExperiencia, String tono) {
        super(nombre, añosExperiencia);
        this.tono = tono;

    }

    public String getTono() {
        return tono;
    }

    public double recalcularSalario(double salario) {
        return ICantante.super.recalcularSalario(salario);
    }
}

File: banda/IvanRC-banda-Kotlin/src/main/kotlin/modelsBandaMusicos/Cantante.kt
package modelsBandaMusicos

class Cantante(nombre: String, salario: Double = 1500.00,añosExperiencia: Int, override val tono: String):
    Musico(nombre, salario, añosExperiencia),ICantante
{
    override fun respirar(){
        println("Cantante: respiro como cantante...")
    }
}